docs/root/configuration/listeners/listener_filters/proxy_protocol.rst

```diff
@@ -29,10 +33,21 @@ If there is a protocol error or an unsupported address family
 Statistics
 ----------
 
-This filter emits the following statistics:
+This filter emits the following general statistics:
+
+.. csv-table::
+  :header: Name, Type, Description
+  :widths: 1, 1, 2
+
+  downstream_cx_proxy_proto_error, Counter, Total number of connections with proxy protocol errors
+
+The filter also emits the statistics rooted at *downstream_cx_proxy_proto.<version>.* for each matched proxy protocol version.
+Proxy protocol versions include ``v1``, ``v2``, and ``not_found``.
 
 .. csv-table::
   :header: Name, Type, Description
   :widths: 1, 1, 2
 
-  downstream_cx_proxy_proto_error, Counter, Total proxy protocol errors
+  allowed, Counter, Total number of connections allowed
+  denied, Counter, Total number of connections rejected due to :ref:`disallowed_versions <envoy_v3_api_field_extensions.filters.listener.proxy_protocol.v3.ProxyProtocol.disallowed_versions>`.
```

Comment:
Will it count when ``allow_requests_without_proxy_protocol=true``?

Reply:
`not_found.allowed` will be incremented when `allow_requests_without_proxy_protocol=true` and `not_found.error` will be incremented when `allow_requests_without_proxy_protocol=false`. This is to maintain in sync with pre-existing metric `downstream_cx_proxy_proto_error`. See the other discussion https://github.com/envoyproxy/envoy/pull/32861#discussion_r1522304738

Added this info to the RST doc.